source/agora/common/API.d

```diff
@@ -106,4 +103,16 @@ public interface API
     ***************************************************************************/
 
     public PublicConfig getPublicConfig();
+
+    /***************************************************************************
+
+        Returns:
+            bool
+
```

Comment:
This should document what the function does, and also the value of the boolean return value.

Reply:
Modified.

Reply:
@AndrejMitrovic,
Check Please~